molgenis-metadata-manager/src/main/frontend/src/store/mutations.js

```diff
@@ -29,5 +27,19 @@ export default {
   },
   [UPDATE_EDITOR_ENTITY_TYPE] (state, update) {
     state.editorEntityType[update.key] = update.value
+  },
+  [SET_SELECTED_ATTRIBUTE_ID] (state, selectedAttributeId) {
+    state.selectedAttributeId = selectedAttributeId
+  },
+  /**
+   * Alert mutations
+   * @param alert Object containing 'type' and 'message' Strings
+   */
+  [CREATE_ALERT] (state, alert) {
```

Comment:
by 'simply' setting (or removing) the error you would only need a single 'set' mutation ( see navigator as example).